tools/local-presubmit

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+# A basic script to locally semi-simulate the build and test
+# operations performed by our continuous integration service. Note
+# that this does not do a clean and reinstall of npm modules and
+# sigh/bazel artifacts.
+
+MAG="\033[95m"
+END="\033[0m"
+
+status() {
+    echo -e "$MAG$1$END"
+}
+
+ROOT=$(dirname $0)/..
+cd $ROOT
+status "Testing all of the things."
```

Comment:
Consider changing to "Testing all of the things (except shells/android and javaharness)"

Reply:
Done.